src/scheduler.h

```diff
@@ -7,11 +7,12 @@
 
 //
 // NOTE:
-// boost::thread / boost::chrono should be ported to std::thread / std::chrono
+// boost::thread should be ported to std::thread
```

Comment:
in commit d0ebd93270758ea97ea956b8821e17a2d001ea94:

Actually `boost::thread` is no longer needed an can be replaced with `std::thread` in this file and the scheduler tests. I know that the thread group is still boost, but the scheduler doesn't care about that.